samples/subsys/zbus/benchmark/sample.yaml

```diff
@@ -1,11 +1,14 @@
 sample:
   name: Benchmark
+common:
+  build_only: true
```

Comment:
this sample builds and runs already on many platforms, build_only does not make sense if we have harness already set to console and we are able to eval the output in sumulator and verify it runs as expected.